### ISSUE TITLE
Multiple Servers Port Issue (Fix provided here)

### ISSUE THREAD

**Issue by Skila0** [citizenfx/txAdmin#102]
Apologies Im new to the whole github stuff. 
Anyways If you own multiple servers and are trying to get them all working on the same machine you must set them up with the convars provided on the github page. However there is a slight bug. If you set them up with different ports, when you go to close the bat and start it back up, it reverts to the default port. The issue is that txAdmin itself is not putting in the Port Convar in the starter bat, so whenever you go to start that specific server it will not work.
Long story short, Edit your starter bat and put this in it = +set txAdminPort 00000 <-- this being your designated port.

**Comment by Rhydium** [comment]
That's not a bug

**Comment by Skila0** [comment]
Well how does one go about doing multiple servers if they cannot launch their servers?

**Comment by Rhydium** [comment]
You can just edit the .bat script that gets generated if you don't want to put the convar everytime.
It isn't a feature that it automatically saves the port that you used.

**Comment by Skila0** [comment]
Well yes and no. If someone is already running their server on that one port then the setup wont work because their server is already on. Some people cant actually risk too much downtime. Anyways I would suggest having all servers through the one txadmin port, but using their own fivem ports etc. Saves setup time & extra hassle. It would also expand the usage of the panel

**Comment by Rhydium** [comment]
Yes I understand, this is something that's requested often (to have control over multiple servers in one txAdmin instance) but I believe it's still not on the roadmap for txAdmin. @tabarra

**Comment by Skila0** [comment]
Fair point. Lets see what the big man would say.

**Comment by ChiningGlitch** [comment]
@tabarra Can you please help me setup txAdmin on a different port but with the same IP.

**Comment by ChiningGlitch** [comment]
or even if you could make this a feature in an upcoming update if its easy to code. We should be able to change the port.

**Comment by tabarra** [comment]
@ChiningGlitch  No, please use our discord server for that.  
![Supporthub](https://media.discordapp.net/attachments/697102099892404344/697102946885697607/Tabarra_Clapback_2_Electric_Boogaloo.jpg?width=707&height=684)

**Comment by Icasafibe** [comment]
@tabarra Discord? Can i please have a link, Thanks

**Comment by tabarra** [comment]
@Icasafibe please be kind and read the README.

**Comment by Rhydium** [comment]
@tabarra maybe it's better if you close this issue ;)

**Comment by tabarra** [comment]
I feel like nobody uses the `.bat` anyways.  
It was done like that more as a "familiarity anchor" for people trying txAdmin for the first time.  
Today, the artifacts don't come with that `run.cmd` anymore, so that became kinda irrelevant.  
Since people might fall in here after googling this, I will add the following `!dual` bot command as reference:  

### How To:
If you want to run two **txAdmin** at the same time, you will need to use convars to set a new profile and port.
If on windows, create a `server2.bat` file inside the FXServer folder and add:
```bat
@echo off
FXServer.exe +set serverProfile "server2" +set txAdminPort "40125"
pause
```
*(feel free to choose any other name or port)*
